.travis.yml

```
@@ -57,6 +57,10 @@ branches:
   only:
     - master
 
+cache:
+  directories:
+    - /home/travis/tests/datasets
```

```suggestion
    - $HOME/tests/datasets
```
Should we use the Travis default env var here? https://docs.travis-ci.com/user/environment-variables/#default-environment-variables

---

Guess this doesn't matter actually since we aren't using Travis anymore